src/styles/cclw.css

```diff
@@ -0,0 +1,69 @@
+#cclw {
+  --text-h1: 48px;
+  --text-h2: 36px;
+  --text-h3: 20px;
+  --text-h4: 21px;
+  --text-h5: 18px;
+  --text-h6: 16px;
+
+  --font-weight-medium: 540;
+}
+
+#cclw .gradient-container {
```

Comment:
What's the difference between this file and `public/css/cclw/theme.css`? Do we need both?